resources/resource-group-default-credential.js

```diff
@@ -15,51 +15,60 @@ References:
 * [Azure SDK Ref Docs for Resources](https://docs.microsoft.com/en-us/javascript/api/overview/azure/resources)
 
 */
-require('dotenv').config();
+
+require("dotenv").config();
 const subscriptionId = process.env["AZURE_SUBSCRIPTION_ID"];
 const myEmailAlias = process.env["EMAIL-ALIAS"];
 const myAppName = process.env["APP-NAME"];
 
 const { DefaultAzureCredential } = require("@azure/identity");
 const { ResourceManagementClient } = require("@azure/arm-resources");
 
-const resourceCreatedDate = new Date().toISOString();
-const resourceGroupName = `${myAppName}-resource-group`;
-const resourceGroupLocation = "eastus";
+async function createResourceGroup() {
+  try {
+    const resourceCreatedDate = new Date().toISOString();
+    const resourceGroupName = `${myAppName}-resource-group`;
+    const resourceGroupLocation = "eastus";

```

Comment:
across the files, your changes are inconsistent when you move variables out of the function or try/catch and when you do. what is your reasoning here?

Reply:
@colawwj Please help to change it back. Thanks